extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -783,7 +783,7 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     if (window.location.hash.startsWith('#settings')) {
       const settingsBtnContainer = $(this.sel.settingsBtnContainer);
       if (settingsBtnContainer.length && !settingsBtnContainer.find('#fc_settings_btn').length) {
-        settingsBtnContainer.children().last().before(this.factory.btnSettings('gmail')); // xss-safe-factory
+        settingsBtnContainer.children().last().before(this.factory.btnSettings('gmail'));
```

Comment:
let's add back `// xss-safe-factory` here and add `.before` to `validateTypeScriptLine`. As now we have `.after` there, but not `.before`

Reply:
Yes, will do - thank you!